### PR TITLE
Add title component

### PR DESCRIPTION
**Pull request by nowak-pawel-ds**
Please verify names of atributes on title component

### CODE REVIEW

File: ui.apps/src/main/jcr_root/apps/bulma/components/title/_ws_dialog/.content.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+               sling:resourceType="wcm/dialogs/components/radio/option"
+               label="Title"
+               value="title"
+               selected="true"/>
+        <subtitle jcr:primaryType="nt:unstructured"
+                  sling:resourceType="wcm/dialogs/components/radio/option"
+                  label="Subtitle"
+                  value="subtitle"/>
+      </type>
+      <elementtype jcr:primaryType="nt:unstructured"
```
Comment (paweljozwicki) [inline ui.apps/src/main/jcr_root/apps/bulma/components/title/_ws_dialog/.content.xml]: To simplify this, maybe we could add Paragraph to heading levels? Maybe with some rename but I'm not sure what name will suits best. E.g. on confluence we have `text style` with `Normal text, Heading 1 and so on`. In terms of that we have also `size` I'm not sure if `text size` is the best name but maybe you will have a better idea.

File: ui.apps/src/main/jcr_root/apps/bulma/components/title/_ws_dialog/.content.xml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      jcr:primaryType="nt:unstructured"
+      sling:resourceType="wcm/dialogs/components/tabs">
+    <generalTab
+        jcr:primaryType="nt:unstructured"
+        sling:resourceType="wcm/dialogs/components/tab"
+        label="General">
+      <type jcr:primaryType="nt:unstructured"
+            sling:resourceType="wcm/dialogs/components/radio"
+            name="type"
+            label="Title/subtitle"
```
Comment (paweljozwicki) [inline ui.apps/src/main/jcr_root/apps/bulma/components/title/_ws_dialog/.content.xml]: Maybe rename to `Type`. There are options title and subtitle so it should be easy to understand

Comment (nowak-pawel-ds) [reply ui.apps/src/main/jcr_root/apps/bulma/components/title/_ws_dialog/.content.xml]: Done

File: ui.apps/src/main/jcr_root/apps/bulma/components/title/_ws_dialog/.content.xml (hunk 1 above)
Comment (nowak-pawel-ds) [reply ui.apps/src/main/jcr_root/apps/bulma/components/title/_ws_dialog/.content.xml]: Paragraph was moved to h1...h6 options as heading element

**Review by paweljozwicki — APPROVED** [review]

**Pull request merged**